resources/osl_authorized_keys.rb

```diff
@@ -0,0 +1,44 @@
+provides :osl_authorized_keys
+resource_name :osl_authorized_keys
```

Comment:
Are these needed when they are the same as the resource filename?

Reply:
Im not sure, ill check

Reply:
If I remove this the implies name is `osl_resources_<RESOURCE NAME>`

Reply:
Ah, good point, I missed that.